doc/src/manual/handling-operating-system-variation.md

```diff
@@ -3,17 +3,18 @@
 When writing cross-platform applications or libraries, it is often necessary to allow for
 differences between operating systems. The variable `Sys.KERNEL` can be used to handle such
 cases. There are several functions in the `Sys` module intended to make this easier:
-`isunix`, `islinux`, `isapple`, `isbsd`, and `iswindows`. These may be used as follows:
+`isunix`, `islinux`, `isapple`, `isbsd`, `isfreebsd`, and `iswindows`. These may be used
```

Comment:
should we update this? Maybe just say “such as these examples” and let people tab complete to find them all.

Reply:
Yeah, listing a few as examples rather than an exhaustive list would be good.